test/puppetlabs/i18n/core_test.clj

```diff
@@ -37,6 +47,22 @@
     (with-user-locale eo
       (is (= welcome_eo (tru welcome_en))))))
 
+(deftest test-trun
+  (testing "trun with no user locale"
+    (is (= one_bottle_en (trun one_bottle n_bottles 1)))
+    (is (= six_bottle_en (trun one_bottle n_bottles 6))))
+  (testing "trun in German"
+    (with-user-locale de (is (= one_bottle_de (trun one_bottle n_bottles 1))))
+    (with-user-locale de (is (= six_bottle_de (trun one_bottle n_bottles 6)))))
+  (testing "trun in Esperanto"
+    ;; We use Esperanto as our test locale
+    (with-user-locale eo (is (= one_bottle_eo (trun one_bottle n_bottles 1))))
+
+;; this isn't working (how should properties file be formatted?
+;;    (with-user-locale eo (is (= one_bottle_eo (trun one_bottle n_bottles 7))))
```

Comment:
I can remove the test.  I couldn't figure out how the properties file should be formatted for pluralization.   It just seemed worth calling out?